README.md

```diff
@@ -155,6 +155,24 @@ ninja -C build swappy-pot
 
 See the [meson documentation](https://mesonbuild.com/Localisation.html) for details.
 
+### Build for Ubuntu
```

Comment:
This is very docker specific, not Ubuntu specific. I don't think it should live in the README file. I would prefer to have this in a `contrib/ubuntu` folder.